src/TypedEnv.purs

```diff
@@ -50,20 +50,28 @@ type VariableFlipped ty name = Variable name ty
 infixr 5 type VariableFlipped as <:
 
 -- | An error that can occur while reading an environment variable
-data EnvError = EnvLookupError String | EnvParseError String
+data EnvError = EnvLookupError String | EnvParseError String | EnvErrors (Array EnvError)
```

Comment:
very useful feature!

but why not just change the parser return type to from `EnvError` to `Array EnvError`

without changing `EnvError`

Reply:
Good suggestion, @srghma. I'm going to merge this change as-is but will follow up with your recommendation.